import exceptions
import fib

print(exceptions.greet("Student"))
